themes/aframe/layout/index.ejs

```diff
@@ -35,6 +35,6 @@
       Get Started
     </a>
 
-    <a class="credits borderless-link" href="http://mozvr.com/" rel="external"></a>
+    <div class="sponsor"><div class="message">Supported by</div><a class="mozilla-logo" href="http://mozilla.com/" rel="external"></a></div>
```

Comment:
if you can, I'd suggest using [Schema.org Microdata HTML](http://schema.org/Organization). it could look something like this:

```html
<p class="sponsor" itemscope itemprop="organization" itemtype="http://schema.org/Organization">
  Supported by
  <span itemprop="sponsor" itemscope itemtype="http://schema.org/Organization">
    <a class="sponsor__mozilla" itemprop="url" href="https://www.mozilla.org/" rel="external">
      <span itemprop="name">Mozilla</span>
    </a>
  </span>
</p>
```